docs/feature-flagging/feature-gates.md

```diff
@@ -32,7 +32,7 @@ user_attributes = {
     device: user.device
 }
 
-if (eppoClient.getStringAssignment(user.id, 'new-checkout-page', user_attributes) == 'on'):
+if (eppoClient.getStringAssignment(user.id, 'new-checkout-page', 'off', user_attributes) == 'on'):
```

Comment:
Feels like this makes more sense as a boolean flag:

```suggestion
if (eppoClient.getBoolAssignment(user.id, 'new-checkout-page', false, user_attributes)):
```

Reply:
I still think we should poll the larger group of engineers about these method signatures. This call looks so weird to me to have the default where it is and subject stuff separated. I'd expect something like:
```py
eppoClient.getStringAssignment('new-checkout-page', user.id, user_attributes, 'off' )
```
Of course not all languages make function overloading easy, so my next choice would be:
```py
eppoClient.getStringAssignment('new-checkout-page', 'off', user.id, user_attributes)
```